### ISSUE TITLE
YAML Exception

### ISSUE THREAD

**Issue by maacl** [ad-si/TaskLite#1]
I get the below when trying to run `tasklite version` after install using stack.

```
YAML exception:
Yaml file not found: /home/mac/.config/tasklite/config.yaml
```

What does a minimal config.yaml have to look like? I have searched the repo but have not been able to find any documentation.

**Comment by orlandoafig** [comment]
dataDir: ''
bodyStyle: Just (Vivid,White)
dateWidth: 10
idStyle: Just (Vivid,Green)
tagStyle: Just (Vivid,Blue)
bodyClosedStyle: Just (Vivid,Black)
priorityStyle: Just (Vivid,Magenta)
closedStyle: Just (Dull,Black)
headCount: 20
dueStyle: Just (Vivid,Yellow)
bodyWidth: 10
maxWidth: 120
progressBarWidth: 24
prioWidth: 4
overdueStyle: Just (Vivid,Red)
dateStyle: Just (Dull,Black)
utcFormat: YYYY-MM-DD H:MI:S
tableName: tasks
dbName: main.db

**Comment by orlandoafig** [comment]
In the `tasklite-core` folder:

`$ stack repl source/Config.hs`

Then:
```
*Config> import Data.Text.Encoding
*Config Data.Text.Encoding> writeFile "config.yaml" $ decodeASCII $ Data.Yaml.encode $ toJSON defaultConfig
```

**Comment by maacl** [comment]
Thanks. Maybe this should be generated per default.

**Comment by ad-si** [comment]
Thanks for your thoughts on this!

This was fixed by 7407f87.
It now automatically creates a default config file, and the documentation contains a link to an example config file (http://tasklite.ad-si.com/installation/cli.html#configuration)
